packages/suite-web/e2e/tests/suite/passphrase-disabled.test.ts

```diff
@@ -1,68 +0,0 @@
-// @group_passphrase
-// @retry=2
-
-const DEFAULT_STANDARD_WALLET_LABEL = 'Standard wallet';
-const DEFAULT_HIDDEN_WALLET_LABEL = 'Hidden wallet #';
-
-describe('Suite switch wallet modal', () => {
-    beforeEach(() => {
-        cy.viewport(1440, 2560).resetDb();
-        cy.task('startBridge');
-    });
-
-    it('passphrase_protection: false', () => {
-        cy.task('startEmu', { wipe: true });
-        cy.task('setupEmu', { passphrase_protection: false });
```

Comment:
is it obsolete? it can just happen that user connects a device with passphrase protection turned off

Reply:
right, this test indeed is obsolete as it is. I would still suggest reworking it to testing something like this

1. add a passphrase wallet, 
2. turn off passphrase (need to be done outside of suite, I see that in device settings this option has disappeared once it is turned on). 
3. try to access account of a passphrase wallet and see that this is well handled 
![Uploading image.png…]()